### ISSUE TITLE
Error @ runtime: Error opening file - index.html: No such file or directory

### ISSUE THREAD

**Issue by bitnom** [ThomasTJdev/nim_nimble_gui#5]
In the nimblegui window is displayed only the following error:
```
Error opening file //tmp/nimble_3196/githubcom_ThomasTJdevnimnimblegui/src/index.html: No such file or directory
```

_Nim Compiler Version 1.2.0 [Linux: amd64]_
_nimblegui version: 0.2.1_

![error screenshot](https://i.imgur.com/hPvmVs0.png)

**Comment by ThomasTJdev** [comment]
Are you cloning and compiling manually or using `nimble install`?

**Comment by ThomasTJdev** [comment]
@juancarlospaco, can you assist me here? I have added `nimblegui` in the `bin`-specification in the nimble-file, so when installing with `nimble install nimblegui` it can't find the index.html. I'm using the `currentHtmlPath`. Should I do a staticRead and include the `index.html` as a `const` or what would you suggest?

**Comment by juancarlospaco** [comment]
Yeah, staticRead should be the best, nimble implicitly builds the binaries on `/tmp/`.

I think that `currentHtmlPath` should be removed, is confusing and it wont help much,
and maybe is better if each one uses their own way of pointing out the path,
sometimes it works on your PC and sometimes it wont on someone else PC.

I added `{.deprecated.}` to `currentHtmlPath` so people dont use it much, not going to remove it tho.
:)

**Comment by ThomasTJdev** [comment]
Fix 67f0e16
